C#/C#.sublime-syntax

```diff
@@ -1361,29 +1363,32 @@ contexts:
             2: punctuation.separator.cs
             3: punctuation.section.brackets.end.cs
             4: keyword.operator.pointer.cs
-    - match: \(
-      scope: punctuation.section.group.begin.cs
+    - match: (?=\()
       branch_point: tuple_or_group
       branch:
-        - group
-        - tuple
+        - - match: \(
+            scope: punctuation.section.group.begin.cs
+            set: group
+        - - match: \(
+            scope: punctuation.section.sequence.begin.cs
+            set: tuple
     - match: \{
       scope: punctuation.section.block.begin.cs
       set:
         - match: \}
           scope: punctuation.section.block.end.cs
           pop: true
         - include: code_block_in
-    - match: (?=\}|\)|>|\]|,|{{reserved}})
+    - match: (?=[]}>,]|{{reserved}})
```

Comment:
This regex irritated me for a second. Maybe readability can be improved?

Reply:
I don't think lines need to be split. There are lots of such patterns everywhere. Maybe the `]` at first position is a quite nifty trick to prevent escaping, but otherwise its a normal character class alternated by a variable.

Reply:
looking at the regex pattern in ST (using PackageDev's highlighting) compared to GitHub's standard YAML highlighting makes quite a big difference to readability also. Happy to gather more opinions and split into two match patterns even if it could help ease of maintenance.